app/views/planning_applications/steps/_review.html.erb

```diff
@@ -3,7 +3,7 @@
   <ul class="app-task-list__items">
     <li class="app-task-list__item">
       <span class="app-task-list__task-name">
-        <% if @planning_application.can_review_assessment? && current_user.reviewer? %>
+        <% if @planning_application.awaiting_determination? && current_user.reviewer? %>
```

Comment:
Does it make sense to just drop the `current_user.reviewer?` part from this instead?

as we have

```
  def can_review_assessment?
    awaiting_determination? && Current.user.reviewer?
  end
  ```
  
  But yeah no need for both (`@planning_application.can_review_assessment? && current_user.reviewer?`) conditional checks

Reply:
I could go either way. I think to me it wasn't obvious  that` can_review_asessment?` checked `current_user.reviewer?` and `awaiting_determination?`

By making them the same I think it makes it clearer what's going on and might allow further refactor in another PR.

Reply:
I am going to introduce a fourth branch. So I want to make it as easy as possible.